sql/references.sql

```diff
@@ -48,4 +48,14 @@ ADD CONSTRAINT fk_db_order_transactions_debit_order_id FOREIGN KEY (debit_order_
 -- changeset devwasabi2:add-fk-constraints-db-order-transaction-transaction
 ALTER TABLE debit_order_transaction
 ADD CONSTRAINT fk_db_order_transactions_transaction_id FOREIGN KEY (transaction_id) REFERENCES transaction(transaction_id);
--- rollback ALTER TABLE debit_order_transaction DROP CONSTRAINT fk_db_order_transactions_transaction_id
\ No newline at end of file
+-- rollback ALTER TABLE debit_order_transaction DROP CONSTRAINT fk_db_order_transactions_transaction_id
+
+-- changeset ryanbasiltrickett:add-fk-constraints-account-delegation-account
+ALTER TABLE account_delegation
+ADD CONSTRAINT fk_account_delegation_account_id FOREIGN KEY (account_id) REFERENCES account(account_id);
+-- rollback ALTER TABLE account_delegation DROP CONSTRAINT fk_account_delegation_account_id
+
+-- changeset ryanbasiltrickett:add-fk-constraints-account-delegation-delegated-account
+ALTER TABLE account_delegation
+ADD CONSTRAINT fk_account_delegation_delegated_account_id FOREIGN KEY (delegated_account_id) REFERENCES account(account_id);
+-- rollback ALTER TABLE account_delegation DROP CONSTRAINT fk_account_delegation_delegated_account_id
```

Comment:
Maybe make it easier and just change the main table and we will just recreate the db from scratch